script.js

```diff
@@ -0,0 +1,175 @@
+/* Створити функцію-конструктор Людини. Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, національність,
+країна, список країн для подорожей. Створити універсальні функції - знайомство, прокидатися, засинати й список країн, які Людина хоче відвідати.
+Роботу цих функцій можна виводити через console.log. Але в кожній з функцій повинно використовуватись якомога більше
+характеристик конкретної Людини на якій ця функція викликається (підказка: тут треба використатии call/apply/bind) */
+
+function Person(name, age, sex, nationality, country, travelList) {
+  this.name = name;
+  this.age = age;
+  this.sex = sex;
+  this.nationality = nationality;
+  this.country = country;
+  this.travelList = travelList;
+}
+
+const oleg = {
+  name: "Oleg",
+  age: 38,
+  sex: "man",
+  nationality: "ukrainian",
+  country: "Ukraine",
+  travelList: ["USA", "Canada", "Spain", "Portunal", "Italy"],
+};
```

Comment:
It should be an instance of `Person`
```suggestion
const oleg = new Person("Oleg", 38, "man", "Ukrainian", "Ukraine", ["USA", "Canada", "Spain", "Portugal", "Italy"],);
```